docs/release_notes.md

```diff
@@ -1,5 +1,46 @@
 # Release Notes
 
+## Version 1.4.0
+
+### Core
+
+- Implemented a new solver to make muscles behave more compact: AdnGlue solver.
+- Implemented Soft Constraints.
+- Added parallelization to Distance constraints.
+- Added parallelization to Shape Preservation constraints.
+- Added Hard Constraints to theAdnFat solver.
+- Added control to the minimum parallelizable chunk size via environment variable `MIN_PARALLEL_SIZE`.
+- Increased the Logger verbosity in Debug level.
```

Comment:
Should we expose this improvement to the user? I mean, the user won't be able to see the debug logs, right? Maybe we could say "Improved logger verbosity to help with troubleshooting" or something like that.

Reply:
Mmm the user would be able to see debug logs by changing the environment variable. What we should do, probably, is to explain how to modify it. I will add a section in FAQ for this (not in this issue). If that is ok with you, I would leave this line as it is. Please, let me know!

Reply:
Agreed, it's an improvement in my opinion

Reply:
Okay, makes totally sense, thank you!